pkg/scheduler/ugm/manager_test.go

```diff
@@ -1312,11 +1367,11 @@ func createUpdateConfigWithWildCardUsersAndGroups(user string, group string, wil
 	return conf
 }
 
-func createUpdateConfig(user string, group string) configs.PartitionConfig {
-	return createConfig(user, group, "memory", "10", 50, 5)
+func createUpdateConfig(user string, groups []string) configs.PartitionConfig {
+	return createConfig(user, groups, "memory", "10", 50, 5)
 }
 
-func createConfig(user string, group string, resourceKey string, resourceValue string, mem int, maxApps uint64) configs.PartitionConfig {
+func createConfig(user string, groups []string, resourceKey string, resourceValue string, mem int, maxApps uint64) configs.PartitionConfig {
```

Comment:
Do we really need this change as part of this pr? 

If no, can we take it up in different jira? we can change user too.

Reply:
To keep this PR pure simple,  removed the refactor code.